### PR TITLE
Server

### PR DESCRIPTION
**Pull request by MichaelChan20**
Added flask server

### CODE REVIEW

**Comment by Remi-Lejeune** [comment]
> Added flask server

It d be nice if you could explain a bit more next time :)

**Review by Remi-Lejeune — APPROVED** [review]
Good job! Everything seems to be working

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+FROM python:3.11-slim
+
+WORKDIR /model-service
+
+# Copy the files into the container
+COPY poetry.lock pyproject.toml /model_service/app.py /model-service/
+
+# Install Poetry and dependencies
+RUN pip install poetry \
+    && poetry config virtualenvs.create false \
```
Comment (sr1998) [inline Dockerfile:11]: Why is this needed?

File: model_service/app.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    
+    res = {
+        "result": labels[0].tolist(),
+        "probability": probabilities[0].tolist(),
+        "url": data 
+    }
+    print(res)
+    return jsonify(res)
+
+#TODO: either download from public link or mount files
```
Comment (sr1998) [inline model_service/app.py:78]: are we doing this?

**Review by sr1998 — COMMENTED** [review]
Just some questions to discuss (PR already done)

File: model_service/app.py (hunk 2 above)
Comment (MichaelChan20) [reply model_service/app.py:78]: Yes this we already do, we download from gdrive and this TODO can be removed.

File: Dockerfile (hunk 1 above)
Comment (MichaelChan20) [reply Dockerfile:11]: This was mainly for if you run the containe standalone to ensure the dependencies are there. Im not sure if its needed once composed.
